Makefile

```diff
@@ -99,7 +99,7 @@ build: ## Compiles the insights operator
 
 .PHONY: build-debug
 build-debug: ## Compiles the insights operator in debug mode
-	go build $(GO_BUILD_FLAGS) -gcflags="all=-N -l" \
+	go build -gcflags="all=-N -l" \
```

Comment:
I know it's not used, but are we safe to remove it? Most of the tests are passing so probably OK to remove.....

Reply:
Yup, this variable was removed from the Makefile and it's not used in `build`, so it doesn't make sense to have it in `build-debug`

Reply:
OK. Hopefully there's no other environment using it....